client/webserver/locales/en-us.go

```diff
@@ -217,7 +218,8 @@ var EnUS = map[string]string{
 	"Maximum Possible Swap Fees":  "Maximum Possible Swap Fees",
 	"max_fee_conditions":          "This is the most you would ever pay in fees on your swap. Fees are normally assessed at a fraction of this rate. The maximum is not subject to changes once your order is placed.",
 	"wallet_logs":                 "Wallet Logs",
-	"accelerate_order":            "Accelerate Order",
+	"Accelerate_Order":            "Accelerate Order",
+	"accelerate_order":            "accelerate order",
```

Comment:
Interesting issue, and not the first time we've had to do this.

I'm considering something like:

```
	"Accelerate_Order":            ":title:accelerate_order",
	"accelerate_order":            "accelerate order",
```

With something like this:

```diff
diff --git a/client/webserver/template.go b/client/webserver/template.go
index 554ecf3c..e30e0b24 100644
--- a/client/webserver/template.go
+++ b/client/webserver/template.go
@@ -12,6 +12,8 @@ import (
        "strings"
 
        "decred.org/dcrdex/client/webserver/locales"
+       "golang.org/x/text/cases"
+       "golang.org/x/text/language"
 )
 
 // pageTemplate holds the information necessary to process a template. Also
@@ -27,6 +29,7 @@ type templates struct {
        fs           fs.FS // must contain tmpl files at root
        reloadOnExec bool
        dict         map[string]string
+       titler       cases.Caser
 
        addErr error
 }
@@ -44,6 +47,7 @@ func newTemplates(folder, lang string) *templates {
                t.addErr = fmt.Errorf("no translation dictionary found for lang %q", lang)
                return t
        }
+       t.titler = cases.Title(language.MustParse(lang))
 
        if embedded {
                t.fs = htmlTmplSub
@@ -77,6 +81,12 @@ func (t *templates) translate(name string) (string, error) {
                                return "", fmt.Errorf("warning: no translation text for key %q", key)
                        }
                }
+               if titleKey := strings.TrimPrefix(replacement, ":title:"); titleKey != replacement {
+                       toTitle, found := t.dict[titleKey]
+                       if found {
+                               replacement = t.titler.String(toTitle)
+                       }
+               }
                rawTmpl = bytes.ReplaceAll(rawTmpl, token, []byte(replacement))
        }
```

Reply:
A different approach would use that `:title:` prefix in the .tmpl. That might be more versatile.

Reply:
> A different approach would use that :title: prefix in the .tmpl.

Yeah, I think this works. But in another work, we might need to have a consistent key format for translations.